src/Resource/NullResource.php

```diff
@@ -0,0 +1,14 @@
+<?php namespace League\Fractal\Resource;
+
+class NullResource extends ResourceAbstract
+{
+    /**
+     * Get the data.
+     *
+     * @return mixed
+     */
+    public function getData()
+    {
+        return null;
```

Comment:
Right, but sometimes you want to make your intentions specifically known. This seems like it could be one of those times.